### ISSUE TITLE
Mafia [54540009]

### ISSUE THREAD

**Issue by Blackbird88** [Cxbx-Reloaded/game-compatibility#419]
### Description
Game gets to menu and even in-game in some modes. Freezes when menu is left alone for a while and video tries to play.

### Dumps 
[XBE Dump](https://gist.github.com/Blackbird88/efc47b61a95f35b12629ad12701f9bae)
[KrnlDebug](https://gist.github.com/Blackbird88/b4e4ec3809d4fb8b53ff4011edff5033)

### Status
**Current behavior:**
In-game with broken graphics and no audio.

### Screenshots
![image](https://user-images.githubusercontent.com/4414625/31861452-c03f4448-b72d-11e7-8c72-982d873872dd.png)
![image](https://user-images.githubusercontent.com/4414625/31861456-cb946b02-b72d-11e7-9108-6460bb768e44.png)

**Comment by ObiKKa** [comment]
Oh, just to remind everybody, a DRM-free form of the PC version of this game was released digitally on [GOG's online store](https://www.gog.com/news/release_mafia) 3 days ago but with an edited soundtrack, thus removing all licensed music from the original edition. Luckily I've got the original retail box. Phew!
Thus this posits that the equivalent Xbox version may also have the same licensed music tracks.

**Comment by kennethedmonds** [comment]
![capture](https://user-images.githubusercontent.com/17582879/34635830-b31163c4-f25a-11e7-9e78-cca2da56d94a.JPG)

As of Jan 5, 2018, this error shows as it is loading to go in game.

**Comment by ObiKKa** [comment]
Status: This game should now get back in-game for the testers as seen in the video below.
Intro splash screens do show, the menus work. In-game graphics & GUI are very good, and, oh yeah, no more corruption as seen in the old OP above. There are just the blue shading issues on the road over the horizon in any camera view. Framerate is good.
- [Cxbx-Reloaded Xbox Emulator - Mafia Ingame! (3c201d7 + WIP / HW Mode, FXAA)](https://www.youtube.com/watch?v=xnXd73t-voM&list=PL4PD8cV5ypNDPTJbrol7A-5rvuteHWWYj&index=45&t=0s) (John GodGames Emus. Published on Mar 6, 2018. Build came just before Luke's vblank timing fix, though has his QueryPerformanceCounter fix; but the timing here is good. Screen is stretched widescreen.)

**Comment by Blackbird88** [comment]
Indeed. I couldn't get Cxbx to work for longest time so I couldn't check myself (turns out one of the folders in AppData were broken somehow.)
Will update OP.

**Comment by Shideravan** [comment]
The channel Literalmente {game} published a test of this game: 
https://www.youtube.com/watch?v=meyAAYKWL7M

**Comment by Blackbird88** [comment]
Videos now work
![image](https://user-images.githubusercontent.com/4414625/40677923-5f87a6c8-637f-11e8-891a-efc60c715832.png)

**Comment by gandalfthewhite19890404** [comment]
Game does not boot on latest master, I'm going to find regressed commit.

**Comment by Blackbird88** [comment]
Still works for me.

**Comment by ObiKKa** [comment]
A [comment, dated November 20, 2020](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/pull/2033#issuecomment-729022774), under a recent PR 2033, that was opened two weeks ago on November 16, 2020 (And then merged later on Nov 21), showed that the map (Map screen or mini-map HUD icon?) and letters (On a menu screen page?) in this game should already be rendered. The PR, which is about improving viewport calculations, fixed the scaling of the menu and UI-related elements on-screen.

Please retest this game.

**Comment by Blackbird88** [comment]
Both of those were fixed already. I have also finished the game to the end, so it's definitely Playable, however there's lot of random crashes.
